emrun.py

```diff
@@ -1486,7 +1534,7 @@ def run():
     if file_to_serve == '.' or file_to_serve_is_url:
       serve_dir = os.path.abspath('.')
     else:
-     serve_dir = os.path.dirname(os.path.abspath(file_to_serve))
+     serve_dir = file_to_serve if (file_to_serve.endswith('/') or file_to_serve.endswith('\\') or os.path.isdir(file_to_serve)) else os.path.dirname(os.path.abspath(file_to_serve))
```

Comment:
Can you write this as multi-line if /else?   In fact it can be and `elif` + `else` with the existing if block maybe?

Reply:
Sure